doc/user/content/installation/operational-guidelines.md

```diff
@@ -12,10 +12,19 @@ menu:
 ## Recommended instance types
 
 - ARM-based CPU
-- 1:8 ratio of vCPU to GiB memory
```

Comment:
Ported over cloud specific content to the cloud provider specific pages and reference those pages here.

Later on, will remove this page and just have the cloud specific pages.  But, didn't want to expand the scope more than I already did 😄